helpers/renderingInfoScript.js

```diff
@@ -217,9 +217,91 @@ function getMinibarsScript(context) {
   `;
 }
 
+function getSearchFormInputScript(context) {
+  const dataObject = `window.${context.id}Data`;
+  const searchFormInputAddEventListeners = `searchFormInputAddEventListeners${context.id}`;
+  const searchFormInputHideRows = `searchFormInputHideRows${context.id}`;
+  const searchFormInputShowRows = `searchFormInputShowRows${context.id}`;
+  const filterRows = `filterRows${context.id}`;
+
+  return `
+    function ${searchFormInputHideRows}() {
+      ${dataObject}.showMoreButtonElement.style.display = '';
+
+      ${dataObject}.tableElement.querySelectorAll('tbody tr').forEach(function(rowElement, index) {
+        if (index >= (${dataObject}.numberOfRows - ${dataObject}.numberOfRowsToHide)) {
+          rowElement.classList.remove('q-table-state-visible');
+          rowElement.classList.add('q-table-state-hidden');
+        }
+      });
+      ${dataObject}.showMoreButtonElement.textContent = 'Alle ' + ${dataObject}.numberOfRows + ' anzeigen';
+      ${dataObject}.rowVisibilityState = 'hidden';
+    }
+
+    function ${searchFormInputShowRows}() {
+      ${dataObject}.showMoreButtonElement.style.display = 'none';
+
+      ${dataObject}.tableElement.querySelectorAll('tbody tr').forEach(function(rowElement, index) {
+        rowElement.classList.remove('q-table-state-hidden');
+        rowElement.classList.add('q-table-state-visible');
+      });
+      ${dataObject}.showMoreButtonElement.textContent = "Tabelle zuklappen";
+      ${dataObject}.rowVisibilityState = 'visible';
+    }
+
+    function ${filterRows}(filter) {
+      filter = filter.toUpperCase();
+      
+      if (filter.length < 2) return;
+
+      ${dataObject}.tableElement.querySelectorAll('tbody tr .q-table__cell--text').forEach(
+        function(cellElement, index) {
+          txtValue = cellElement.innerText.toUpperCase();
```

Comment:
Using a more expressive variable name should help understanding what you're doing here

Reply:
You're refering to the variable `txtValue` right?